### DISCUSSION TITLE
How do you re-sync after unsync notes?

### DISCUSSION THREAD

**Discussion by chs33** [windingwind/zotero-better-notes#781] [Q&A]
I accidentally hit the unsync in the sync manager,

Is there a way to let BN re-sync with that previously synced markdown notes? or the only way right now is to export to a new notes and set auto-sync and overwrite the old one?

I think this feature also has use case working on multiple devices with sync, which I am not sure BN supported or not currently.

**Comment by windingwind** [comment]
unfortunately, if you unsync them, then they're unsynced - you'll have to export - sync these notes again.

maybe i should add a confirmation for unsync button?

**Reply by windingwind** [reply]
Due to the unavailability of settings and synced ids across devices, it is impossible to automatically sync the same notes between devices. You'll have to manually set up for each device.
